demos/gaze_estimation_demo/cpp_gapi/include/eye_state_estimator.hpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cstdio>
+#include <string>
+
+#include "face_inference_results.hpp"
+
+namespace gaze_estimation {
+class EyeStateEstimator {
+public:
+    void estimate(const std::vector<cv::Mat>& left_eyes_state,
```

Comment:
This should become a separate function (or you can move this function body into `static void run()` directly): `EyeStateEstimator` already diverged with cpp's one and `EyeStateEstimator` is stateless now. Same for others.
This will also remove a need to keep global `Estimators estimators;`.

Reply:
Removed estimators. Moved code to kernels.